packages/components/src/functional-components/FormFieldLabel/tests/snapshot.test.tsx

```diff
@@ -0,0 +1,71 @@
+import { h } from '@stencil/core';
+import KolFormFieldLabelFc from '../FormFieldLabel';
+import { renderFunctionalComponentToSpecPage } from '../../../utils/testing';
+
+describe('KolFormFieldLabelFc', () => {
+	it('should render with label', async () => {
+		const label = 'Test Label';
+		const id = 'test-id';
+		const page = await renderFunctionalComponentToSpecPage(() => <KolFormFieldLabelFc label={label} id={id} />);
+
+		expect(page.root).toMatchSnapshot();
+	});
+
+	it('should render with accessKey and shortKey', async () => {
```

Comment:
Den Case würde ich weglassen, weil immer nur eine der beiden Eigenschaften möglich sein sollte.

Reply:
Dies mag in der WC richtig sein, aber in der FC kann beides gleichzeitig vorkommen und dann sollte klar welches der beiden gewinnt